### ISSUE TITLE
where is the pre-built bartholomew.wasm located

### ISSUE THREAD

**Issue by michelleN** [fermyon/bartholomew#2]
Potentially silly question but where is the pre-built bartholomew.wasm located?

**Comment by radu-matei** [comment]
It should be in the `target/wasm32-wasi/release` directory (the default one for Rust `wasm32-wasi` projects).

**Comment by michelleN** [comment]
Thanks Radu. That makes sense for when I'm building the project myself but this line in the readme makes it seem like there is one I can download hosted somewhere on this github repo:
```
You can also use the pre-built batholomew.wasm or the versioned bindles.
```

Am I understanding this incorrectly?

**Comment by radu-matei** [comment]
Ah, you are correct, that is missing.
In fact, the static file server module (https://github.com/fermyon/bartholomew/blob/main/modules.toml#L2) is missing as well.

**Comment by technosophos** [comment]
This was resolved with the 0.1 release! Yay!
